weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallPage.tsx

```diff
@@ -97,6 +103,33 @@ const CallPageInnerVertical: FC<{
   call: CallSchema;
   path?: string;
 }> = ({call, path}) => {
+  const {loading: viewerLoading, userInfo} = useViewerUserInfo2();
```

Comment:
nit: can you wrap all this up into a single hook: `useRecordCallView` or something that moves most of the implementaiton out of the component?